include/roboteam_ai/world_new/WorldData.hpp

```diff
@@ -97,6 +98,11 @@ class WorldData {
      */
     WorldData &operator=(WorldData &&) = default;
 
+    /**
+     * Does exactly what it says, it sets the internal vectors for the views
+     */
+    void setViewVectors() noexcept;
```

Comment:
shouldn't this function be private?

Reply:
Doesn't really matter, it doesn't fuck with the actual robots so yeah, but I'll make it private.